docs/team/chikchengyao.adoc

```diff
@@ -0,0 +1,72 @@
+= Chik Cheng Yao - Project Portfolio
```

Comment:
Either leave as John Doe , or prepend all docs with a line that indicates this being a lorem ipsum equivalent